addons/website/static/src/scss/website.scss

```
@@ -406,6 +406,12 @@ $-header-nav-link-height: $nav-link-height;
     }
 }
 
+hr {
+    border-top: 1px solid $border-color;
```

Users can override BS variables and, by doing so, customise the website design. 
BS provides an useful set of variables that defines the `<hr>` default design (see [here](https://github.com/twbs/bootstrap/blob/1a6fdfae6be09b09eaced8f0e442ca6f7680a61e/scss/_variables.scss#L663)), we should use them to ensure that overriding BS variables achieves the expected result...

```suggestion
    border-top: $hr-height solid $hr-color;
```